packages/components/src/form/FieldSet.tsx

```diff
@@ -0,0 +1,102 @@
+import { cns } from '@stcland/utils'
+
+//*****************************************************************************
+// Interface
+//*****************************************************************************
+interface Props {
+  title?: string;
+  name?: string;
+  label?: string;
+  hidden?: boolean;
+  disabled?: boolean;
+  withPanel?: boolean;
+  children: React.ReactNode;
+  customStyles?: FieldSetStyles;
+}
+
+type FieldSetStyles = Partial<{
+  root: React.HTMLAttributes<HTMLDivElement>['className'];
+  label: React.HTMLAttributes<HTMLDivElement>['className'];
+  title: React.HTMLAttributes<HTMLDivElement>['className'];
+  fieldset: React.HTMLAttributes<HTMLFieldSetElement>['className'];
+}>;
+
+//*****************************************************************************
+// Components
+//*****************************************************************************
+const Fieldset = ({
+  name,
+  label,
+  hidden = false,
+  disabled = false,
+  customStyles,
+  withPanel = false,
+  children,
+  title,
+}: Props) => {
+  const cn: FieldSetStyles = {
+    label: cns('font-bold text-sky-500 text-sm', customStyles?.label),
+    root: cns(
+      'p-4 flex flex-col gap-2 bg-gray-800 text-xl',
+      customStyles?.root
+    ),
+    fieldset: cns(
+      'grid grid-cols-2 gap-2 disabled:cursor-disabled',
+      customStyles?.fieldset
+    ),
+    title: cns(customStyles?.title),
+  }
```

Comment:
@ianmarquez there is a method called `appliedStyles` that can be exported from `@stcland/utils` which does the deep merging with `default` and `customStyles`, so that we don't need to use the `cns` per prop, can we update the occurrences that used this approach?

See example in `Table`, `Button` or `Icon`